packages/model/addon/-private/model.js

```diff
@@ -2,8 +2,10 @@ import { assert, deprecate, warn } from '@ember/debug';
 import EmberError from '@ember/error';
 import EmberObject, { computed, get } from '@ember/object';
 import { dependentKeyCompat } from '@ember/object/compat';
+import { run } from '@ember/runloop';
```

Comment:
we need to specify this import in the external dependencies list in packages/model/index.js